packages/core/src/browser/widget-manager.ts

```diff
@@ -125,7 +150,9 @@ export class WidgetManager {
     }
 
     /**
-     * Try and get the widget.
+     * Try to get the existing widget for the given description.
+     * @param factoryId The widget factory id.
+     * @param options  The widget factory specific information.
```

Comment:
minor: additional whitespaces after the parameter.

Reply:
Fixed